trackintel/preprocessing/positionfixes.py

```diff
@@ -179,17 +125,20 @@ def generate_staypoints(positionfixes, method='sliding',
                 staypoint['id'] = staypoint_id
 
                 # point geometry of staypoint
-                staypoint.geometry = Point(group[name_geocol].x.mean(),
+                staypoint[name_geocol] = Point(group[name_geocol].x.mean(),
```

Comment:
Is there a reason why this should be changed? 
If so, it might be useful to introduce a convention.

Reply:
`staypoint` is a dict and has no `.geometry`